uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml

```diff
@@ -113,7 +112,24 @@ Pane {
 
     implicitHeight: 300 + padding
 
-    padding: 5
+    spacing: 5
+    leftPadding: popupView.spacing
+    rightPadding: popupView.spacing
+
+    title: controller.title
+
+    // Title Header
+    header: Label {
+        textFormat: Text.StyledText
+        text: `<h2>${popupView.title}</h2>`
```

Comment:
❓ why using `popupView.title` and not directly `controller.title`?

Reply:
The full code is this:

```qml
    title: controller.title

    // Title Header
    header: Label {
        textFormat: Text.StyledText
        text: `<h2>${popupView.title}</h2>`
```

`Page` comes with a title property, so it makes sense to populate it with the data so if read it shows something sensible, it also allows the user to override it and see the effect take place.